.gas-snapshot

```diff
@@ -1,2 +1,2 @@
-LightClient_newFinalizedState_Test:testCorrectUpdateBench() (gas: 591187)
-PlonkVerifier_verify_Test:test_verify_succeeds() (gas: 425555)
\ No newline at end of file
+LightClient_newFinalizedState_Test:testCorrectUpdateBench() (gas: 589370)
```

Comment:
Interesting, the gas cost for  updating the state of the light client contract looks reasonable now.

Reply:
not really. the diff (saving in plonk verifier) was offseted and not 1:1 to diff in light client. still need to investigate later.